contracts/ERC7432/ERC7432.sol

```diff
@@ -26,8 +25,7 @@ contract ERC7432 is IERC7432 {
 
     modifier onlyApproved(address _tokenAddress, uint256 _tokenId, address _grantor) {
```

Comment:
It's not necessarily the `grantor`, in some cases we use it with `grantee` as well. Maybe we should rename it for something more neutral?
```suggestion
    modifier onlyApproved(address _tokenAddress, uint256 _tokenId, address _account) {
```